utils/feature-flags.ts

```diff
@@ -7,13 +7,14 @@ declare const window: any
  * Default flag values when LD is not available.
  * NOTE: To disable dashboard tiles, you require an entry in here.
  * Otherwise the tile will get disabled on the frontpage, but not under the dashboard section.
- * These also don't seem to be working when you use a tool to block LaunchDarkly. This needs to be fixed in the future.
  */
 const defaultFlagSet: LDFlagSet = {
   'bcregistry-ui-bca-enabled': false,
   'bcregistry-ui-bus-search-enabled': true,
   'bcregistry-ui-bus-search-beta-chip': false,
   'bcregistry-ui-bus-search-coming-soon-chip': false,
+  'bcregistry-ui-cso-enabled': true,
```

Comment:
Previously, people were complaining that the CSO tile wasn't showing up even with this set to true. 

With the default value change below, this should work as expected.